srcts/src/utils/index.ts

```diff
@@ -144,6 +144,19 @@ function pixelRatio(): number {
   }
 }
 
+function getBoundingClientSizeBeforeZoom(el: HTMLElement): {
+  width: number;
+  height: number;
+} {
+  const rect = el.getBoundingClientRect();
+  // Cast to any because currentCSSZoom isn't in the type def of HTMLElement
+  const zoom = (el as any).currentCSSZoom || 1;
```

Comment:
`currentCSSZoom` was added in typescript release 5.5.2 ([here's the commit](https://github.com/microsoft/TypeScript/commit/e04a27c50427013f9ea768f35c9199d27ecfdaef#diff-dc0eab937d15e62545da3ed7b4f40ad6b24f15dd88fbc6ceda2bfb4ed8356eb0R5085)); worth adding a TODO and hope we catch this in the future.

```suggestion
  // Cast to any because currentCSSZoom isn't in the type def of HTMLElement
  // TODO: typescript >= 5.5.2 added this property to the type definition
  const zoom = (el as any).currentCSSZoom || 1;
```